mteb/evaluation/evaluators/BitextMiningEvaluator.py

```diff
@@ -15,16 +15,20 @@
 logger = logging.getLogger(__name__)
 
 
+DEFAULT_PAIR = [("sentence1", "sentence2")]
+
+
 class BitextMiningEvaluator(Evaluator):
-    def __init__(self, sentences, task_name: str | None = None, subsets=None, **kwargs):
+    def __init__(
+        self,
+        sentences,
```

Comment:
```suggestion
        sentences: Dataset,
```

Reply:
Can't it also be a datasetDict? (generally only type hint when I am 100% sure)